supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorInitializationActor.scala

```diff
@@ -35,20 +32,21 @@ class HtCondorInitializationActor(override val workflowDescriptor: BackendWorkfl
   override def beforeAll(): Future[Unit] = Future.successful(())
 
   /**
-    * Validates runtime attributes for one specific call.
-    *
-    * @param runtimeAttributes Runtime Attributes with already evaluated values.
-    * @return If all entries from runtime attributes section are valid Success otherwise
-    *         Failure with the aggregation of errors.
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
     */
-  override def validateRuntimeAttributes(runtimeAttributes: EvaluatedRuntimeAttributes): Future[ErrorOr[Unit]] = {
+  override def validate(): Future[Unit] = {
     Future {
-      val docker = validateDocker(runtimeAttributes.get(Docker), "Failed to get Docker mandatory key from runtime attributes".failureNel)
-      val failOnStderr = validateFailOnStderr(runtimeAttributes.get(FailOnStderr), FailOnStderrDefaultValue.successNel)
-      val continueOnReturnCode = validateContinueOnReturnCode(runtimeAttributes.get(ContinueOnReturnCode),
-        ContinueOnReturnCodeSet(Set(ContinueOnRcDefaultValue)).successNel)
-      (docker |@| failOnStderr |@| continueOnReturnCode) {
-        (_, _, _)
+      calls foreach { call =>
+        val runtimeAttributes = call.task.runtimeAttributes.attrs
+        val supportedKeys = Set(Docker, FailOnStderr, ContinueOnReturnCode)
```

Comment:
The list of supported keys could be moved to the `object` instead of being re-instantiated every time